gateway-js/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -314,8 +320,12 @@ describe('Supergraph SDL update failures', () => {
 it('Rollsback to a previous schema when triggered', async () => {
   // Init
   mockSupergraphSdlRequestSuccess();
-  mockSupergraphSdlRequestSuccess(getTestingSupergraphSdl(fixturesWithUpdate), 'updatedId-5678');
-  mockSupergraphSdlRequestSuccess();
+  mockSupergraphSdlRequestSuccessIfAfter(
+      'originalId-1234',
+    'updatedId-5678',
+      getTestingSupergraphSdl(fixturesWithUpdate),
+  );
```

Comment:
Indentation consistency nit:
```suggestion
  mockSupergraphSdlRequestSuccessIfAfter(
    'originalId-1234',
    'updatedId-5678',
    getTestingSupergraphSdl(fixturesWithUpdate),
  );
```